File: tasks/prac2_python/task4.py
from math import pi


dlina = int(input("Введите длину прямоугольника: "))
shirina = int(input("Введите ширину прямоугольника: "))

result = dlina * shirina

rad = int(input("Введите радиус: "))

result = pi * rad ** 2
print(round(result, 2))

File: tasks/prac2_js/task4.js
const prompt = require('prompt-sync')();

let dlina = parseFloat(prompt("Введите длину прямоугольника: "));
let shirina = parseFloat(prompt("Введите ширину прямоугольника: "));
let result = dlina * shirina;

console.log(`Площадь прямоугольника ${result}`);

let rad = parseFloat(prompt("Введите радиус круга: "));
result = Math.PI * rad ** 2;
console.log(`Площадь круга: ${result}`);